discordbot/main.py

```diff
@@ -25,6 +29,8 @@ async def load_extensions():
     await bot.load_extension(f"cogs.faq")
     await bot.load_extension(f"cogs.auditlogs")
     # await bot.load_extension(f"cogs.googlecalendar")
+    await bot.load_extension(f"cogs.channels")
+    await bot.load_extension(f"cogs.names")
 
```

Comment:
Add loading admin cog (main had a merge recently)

Reply:
I don't really know what I should do, what should I add or remove?